janitor/functions.py

```diff
@@ -4104,3 +4104,143 @@ def process_text(
     df[column] = getattr(df[column].str, string_function)(*args, **kwargs)
 
     return df
+
+
+@pf.register_dataframe_method
+def fill_direction(
+    df: pd.DataFrame,
+    directions: Dict[Hashable, str],
+    limit: Optional[int] = None,
+) -> pd.DataFrame:
+    """
+    Provides a method-chainable function for filling missing values in selected
+    columns, using the next or previous entry. The columns are paired with the
+    directions in a dictionary. It is a wrapper for ``pd.Series.ffill`` and
+    ``pd.Series.bfill``.
```

Comment:
@samukweku This is what I would be looking for:

```suggestion
    """Provide a method-chainable function for filling missing values in selected columns.
    
    Missing values are filled using the next or previous entry.
    The columns are paired with the directions in a dictionary.
    It is a wrapper for ``pd.Series.ffill`` and ``pd.Series.bfill``.
```

Reply:
@hectormz gotcha.